source/users.rst

```diff
@@ -39,85 +39,59 @@ permissions have been successfully granted.
-|              |             |               | -xtrabackup/2.4/using_ |
-|              |             |               | xtrabackup/privileges. |
-|              |             |               | html>`__               |
-+--------------+-------------+---------------+------------------------+
-| Cluster      | clusterchec | clustercheck  | `User for liveness and |
-| Check        | kuser       |               | readiness              |
-|              |             |               | checks <http://galerac |
-|              |             |               | luster.com/documentati |
-|              |             |               | on-webpages/monitoring |
-|              |             |               | thecluster.html>`__    |
-+--------------+-------------+---------------+------------------------+
-| PMM Client   | monitor     | monitor       | `User for PMM          |
-| User         |             |               | agent <https://www.per |
-|              |             |               | cona.com/doc/percona-m |
-|              |             |               | onitoring-and-manageme |
-|              |             |               | nt/security.html#pmm-s |
-|              |             |               | ecurity-password-prote |
-|              |             |               | ction-enabling>`__     |
-+--------------+-------------+---------------+------------------------+
-| PMM Server   | should be   | pmmserver     | `password to access    |
-| Password     | set via     |               | PMM                    |
-|              | `operator   |               | Server <https://www.pe |
-|              | options <op |               | rcona.com/doc/percona- |
-|              | erator>`__  |               | monitoring-and-managem |
-|              |             |               | ent/security.html#pmm- |
-|              |             |               | security-password-prot |
-|              |             |               | ection-enabling>`__    |
-+--------------+-------------+---------------+------------------------+
-
-Development Mode
-----------------
-
 To make development and testing easier, ``deploy/secrets.yaml`` secrets
-file contains default passwords for PXC system users.
-
-These development mode credentials from ``deploy/secrets.yaml`` are:
-
-============ ========================
-Secret Key   Secret Value
-============ ========================
-root         ``root_password``
-xtrabackup   ``backup_password``
-monitor      ``monitor``
-clustercheck ``clustercheckpassword``
-proxyuser    ``s3cret``
-proxyadmin   ``admin_password``
-pmmserver    ``supa|^|pazz``
-============ ========================
-
-**Warning:** *Do not use the default PXC user passwords in production!*
+file contains default passwords for PXC system users and are mapped with
+key/value pairs. The username is the key and the value is an encoded
+password used to access a server or a system object.
+
+You can decode the value in ``secrets.yaml`` with the following command:
```

Comment:
this doc should help the user to create own secrets file, not decode current

Reply:
OK - that purpose would be a re-write, since we must explain creating a secret object manually or through an editor. Are you just asking to remove the "decode" line and example?

Reply:
if we decided to fully rework document I want to have a document which helps the user to create PXC cluster credentials correctly. not just use passwords from example.